instrumentation-core/spring/spring-core/spring-core.gradle

```
@@ -0,0 +1,27 @@
+ext {
+  minJavaVersionForTests = JavaVersion.VERSION_1_8
+}
+
+group = 'io.opentelemetry.spring.instrumentation'
+version = '0.0.1-SNAPSHOT'
+
+apply plugin: 'java'
+apply from: "$rootDir/gradle/java.gradle"
+
+sourceCompatibility = '8'
+
+dependencies {
+  implementation "org.springframework:spring-context:3.1.0.RELEASE"
+  implementation "org.springframework:spring-webmvc:3.1.0.RELEASE"
+
+  compileOnly deps.opentelemetryApi
```

```suggestion
  implementation deps.opentelemetryApi
```

Manual instrumentation requires this so we make sure the POM reflects it